--- distutils/fcompiler/gnu.py.old	2017-07-14 10:01:26.000000000 +0100
+++ distutils/fcompiler/gnu.py	2017-07-14 10:05:38.000000000 +0100
@@ -333,17 +333,11 @@
         if sys.platform == 'win32':
             c_compiler = self.c_compiler
             if c_compiler and c_compiler.compiler_type == "msvc":
-                if "gcc" in opt:
-                    i = opt.index("gcc")
-                    opt.insert(i+1, "mingwex")
-                    opt.insert(i+1, "mingw32")
-            # XXX: fix this mess, does not work for mingw
-            if is_win64():
-                c_compiler = self.c_compiler
-                if c_compiler and c_compiler.compiler_type == "msvc":
-                    return []
-                else:
-                    pass
+                libs = ['mingw32', 'mingwex', 'gcc', 'gfortran', 'quadmath']
+                if sys.version_info[:2] >= (3, 5):
+                    # https://msdn.microsoft.com/en-us/library/bb531344.aspx#BK_CRT
+                    libs += ['legacy_stdio_definitions', 'iob_hack']
+                return libs
         return opt
 
     def get_target(self):
